resources/example-behaviour.js

```diff
@@ -10,6 +10,24 @@
     });
   }
 
+  // Check whether links to API-docs of used classes actually point to
+  // existing html-files.
+  $('#api-links a').each(function() {
+    var url = this.href,
+        symbol = this.innerHTML;
+    $.ajax({
+      type: 'HEAD',
+      url: url,
+      context: this,
+      error: function() {
+        // we get into the error if either the resource didn't exist
+        // or if HEAD was not allowed (when serving locally via node)
+        // => set the <li> html to the name of the symbol
+        $(this).parent().html(symbol);
```

Comment:
If the text says "Related API documentation" (which I think is more suitable than "Used in this example"), we would want to remove the element if it doesn't exist in the docs.  And then I guess we just hope we never end up with an empty list.